graph/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "hypercerts",
+  "license": "UNLICENSED",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.thegraph.com/deploy/ protocol/hypercerts",
+    "create-local": "graph create --node http://localhost:8020/ protocol/hypercerts",
+    "remove-local": "graph remove --node http://localhost:8020/ protocol/hypercerts",
+    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 protocol/hypercerts",
+    "test": "graph test"
```

Comment:
Probably nice to always run `yarn codegen` when running this command

Reply:
added